src/libs/renderer/src/sdevice.cpp

```diff
@@ -3441,7 +3466,18 @@ HRESULT DX9RENDER::DrawPrimitive(D3DPRIMITIVETYPE dwPrimitiveType, UINT StartVer
 HRESULT DX9RENDER::Release(IUnknown *pObject)
 {
     if (pObject)
-        return pObject->Release();
+    {
+        if (*(void **)pObject == nullptr)
```

Comment:
why do we need this code?

Reply:
This is a kind of debug code that does not harm but should be removed in the future. 
Basically, this check allowed me to detect already released COM objects. And since I'm not done with the feature yet I think it's better to leave it for now.
Of course, if it's not causing issues for anyone...